script/setup

```diff
@@ -5,7 +5,7 @@ case "$(uname -s)" in
 			npm install -g bower grunt-cli && npm install
 		;;
 	*)
-			if [ $(id -u) != "0" ];
+			if [ $(id -u) == "0" ];
```

Comment:
Can you undo this change, it don't seem related to your PR.

Once that completed, we will merge your PR.